Exercise/src/my_string.cpp

```diff
@@ -0,0 +1,80 @@
+#include "my_string.h"
+#include <cstddef>
+
+std::size_t Exercise_1::CMyString::GetStringLength(const char *cpszStringToCount) noexcept {
```

Comment:
for const zero terminated string we use csz, even if it is a ptr we don't add prefix p